regression-test/framework/src/main/groovy/org/apache/doris/regression/Config.groovy

```diff
@@ -79,6 +79,13 @@ class Config {
     public InetSocketAddress metaServiceHttpInetSocketAddress
     public Integer parallel
     public Integer suiteParallel
+
+    public String repo
+    public String testBranch
+    public String serverUrl
+    public String pipelineId
+    public String buildId
+
```

Comment:
Can we put item here in others like webhook?  repo, serverUrl is to general, maybe repoName, xxServerUrl?

Reply:
Yes, it might be a better choice put items here. I will improve this place